pkg/synthetictests/networking.go

```diff
@@ -205,14 +205,12 @@ func testOvnNodeReadinessProbe(events monitorapi.Intervals) []*junitapi.JUnitTes
 			}
 		}
 	}
+	test := &junitapi.JUnitTestCase{Name: testName}
 	if len(failureOutput) > 0 {
-		tests = append(tests, &junitapi.JUnitTestCase{
-			Name: testName,
-			FailureOutput: &junitapi.FailureOutput{
-				Output: failureOutput,
-			},
-		})
-
+		test.FailureOutput = &junitapi.FailureOutput{
+			Output: failureOutput,
+		}
 	}
+	tests = append(tests, test)
```

Comment:
intentionally making this test never fail?  I suggest having a bug reference if that is your goal.

Reply:
Am I misreading something? tests is initialized empty, we only ever add one test to it, and that test has no failure output if successful, and will if it failed. I don't see how we make this never fail.

Reply:
> Am I misreading something? tests is initialized empty, we only ever add one test to it, and that test has no failure output if successful, and will if it failed. I don't see how we make this never fail.

yeah, my mistake.